src/main/java/seedu/taassist/model/uniquelist/UniqueList.java

```diff
@@ -64,6 +64,16 @@ public void addAll(Collection<T> toAddList) {
         Collections.sort(internalList);
     }
 
+    /**
+     * Adds all elements in the {@code toAddList} to the current list.
+     * The elements must not already exist in the list.
+     */
+    public void addAll(UniqueList<T> toAddUniqueList) {
```

Comment:
Is there a need for this method? All current usages seem to be able to cover by `setElements`.